start.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Start script for document-generator
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/document-generator/env
```

Comment:
There is no port configured here? When I said to take it out I mean to just do: `PORT="${DOCUMENT_GENERATION_SERVICE_PORT}"` rather than `PORT="${DOCUMENT_GENERATION_SERVICE_PORT:=4017}"` as the second says assign `DOCUMENT_GENERATION_SERVICE_PORT` to `PORT` however if it is null or doesn't exist, then assign `4017` as the default value.